.github/workflows/actions.yml

```diff
@@ -158,7 +158,7 @@ jobs:
       with:
         submodules: true
     - name: Build in arm32 container
-      uses: uraimo/run-on-arch-action@v2.7.2
+      uses: uraimo/run-on-arch-action@master
```

Comment:
New version **2.8.1** has been released just few hours ago (https://github.com/marketplace/actions/run-on-architecture).
What about using the latest version instead?

Reply:
Thank you for the idea. It is working.